File: i2f-springboot/i2f-springboot-ops-starter/src/main/resources/static/ops/common/OpsSecureKeyPair.js
/**
 * @param publicKey {string}
 * @param privateKey {string}
 * @type {OpsSecureKeyPair}
 * @constructor {OpsSecureKeyPair}
 */
function OpsSecureKeyPair(publicKey,privateKey){
    /**
     *
     * @type {string}
     */
    this.publicKey=publicKey;
    /**
     *
     * @type {string}
     */
    this.privateKey=privateKey;

}

File: i2f-springboot/i2f-springboot-ops-starter/src/main/java/i2f/springboot/ops/common/OpsSecureKeyPair.java
package i2f.springboot.ops.common;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author Ice2Faith
 * @date 2025/11/1 22:25
 * @desc
 */
@Data
@NoArgsConstructor
public class OpsSecureKeyPair {
    protected String publicKey;
    protected String privateKey;

    public OpsSecureKeyPair(String publicKey, String privateKey) {
        this.publicKey = publicKey;
        this.privateKey = privateKey;
    }
}
